keps/sig-architecture/3935-oldest-node-newest-control-plane/kep.yaml

```diff
@@ -0,0 +1,50 @@
+title: Support Oldest Node And Newest Control Plane
+kep-number: 3935
+authors:
+  - "@derekwaynecarr"
+  - "@liggitt"
+owning-sig: sig-architecture
+participating-sigs:
+  - sig-cluster-lifecycle
+  - sig-network
+  - sig-node
+status: provisional
+creation-date: 2023-04-03
+reviewers:
+  - TBD # sig-apps
+  - TBD # sig-auth
+  - TBD # sig-cloud-provider
+  - TBD # sig-scheduling
+  - TBD # sig-storage
```

Comment:
@msau42 @jsafrane - can I add one (or both) of you as reviewers for sig-storage (or can you recommend a delegate?)